templates/system/modules/pwa/themes/base/img/browserconfig.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just curious, is this needed...?

Reply:
This is for smart phones that run Windows.

Reply:
We have covered all smart phones... Apple, Android and Windows :)